indico/modules/rb_new/client/js/modules/admin/EquipmentTypeList.jsx

```diff
@@ -0,0 +1,172 @@
+import {formatters, ReduxDropdownField, ReduxFormField} from 'indico/react/forms';
+import {Param, Plural, PluralTranslate, Singular, Translate} from 'indico/react/i18n';
+import * as adminActions from './actions';
+import * as adminSelectors from './selectors';
+import EditableList from './EditableList';
+
+import './EditableList.module.scss';
+
+
+class EquipmentTypeList extends React.PureComponent {
+    static propTypes = {
+        isFetching: PropTypes.bool.isRequired,
+        equipmentTypes: PropTypes.array.isRequired,
+        features: PropTypes.array.isRequired,
+        actions: PropTypes.exact({
+            deleteEquipmentType: PropTypes.func.isRequired,
+            updateEquipmentType: PropTypes.func.isRequired,
+            createEquipmentType: PropTypes.func.isRequired,
+        }).isRequired,
+    };
+
+    renderItem = ({name, features, numRooms}) => (
+        <List.Content styleName="info">
+            <List.Header>
+                <span styleName="name">{name}</span>
+                {features.map(feat => (
+                    <Popup key={feat.name}
+                           trigger={<Icon name={feat.icon || 'cog'} color="blue" />}>
+                        <Translate>
+                            This equipment type provides the
+                            {' '}
+                            <Param name="name" wrapper={<strong />} value={feat.title} /> feature.
+                        </Translate>
+                    </Popup>
+                ))}
+            </List.Header>
+            <List.Description>
+                {!numRooms ? (
+                    <Translate>Currently unused</Translate>
+                ) : (
+                    <PluralTranslate count={numRooms}>
+                        <Singular>
+                            Available in <Param name="count" wrapper={<strong />}>1</Param> room
+                        </Singular>
+                        <Plural>
+                            Available in
+                            {' '}
+                            <Param name="count" wrapper={<strong />} value={numRooms} />
+                            {' '}
+                            rooms
+                        </Plural>
+                    </PluralTranslate>
+                )}
+            </List.Description>
+        </List.Content>
+    );
+
+    renderForm = fprops => {
+        const {features} = this.props;
+
+        const featureOptions = features.map(feat => ({
+            key: feat.name,
+            value: feat.id,
+            text: feat.title,
+            icon: feat.icon,
+        }));
+
+        return (
+            <>
+                <Field name="name" component={ReduxFormField} as="input"
+                       format={formatters.trim} formatOnBlur
+                       label={Translate.string('Name')}
+                       disabled={fprops.submitting}
+                       autoFocus />
+                {!!featureOptions.length && (
```

Comment:
maybe just use `>0` since you are using it somewhere down below `numEquipmentTypes > 0 && (
`